lib/insights.ts

```diff
@@ -113,9 +112,8 @@ class AlgoliaAnalytics {
   public viewedObjectIDs: (params?: InsightsSearchViewObjectIDsEvent) => void;
   public viewedFilters: (params?: InsightsSearchViewFiltersEvent) => void;
 
-  constructor(options?: any) {
+  constructor({  }: AlgoliaAnalyticsOptions = {}) {
```

Comment:
is there a reason for changing in this PR?

Reply:
Actually it's a stepping stone for this: https://github.com/algolia/search-insights.js/pull/142/files#diff-cc2c0a8a8dfcd80e79e33bc5d480b745R119
I should've left a comment here to avoid confusion.